doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -3311,6 +3311,16 @@ The following is a list of bind types and how they work. Below each bind type is
 
   Module: core
 
+(50) AWY3 (stackable)
```

Comment:
please call this "bind away"

Reply:
we already have a 'bind away' for partyline away status